File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    answer = []
    for i in range(52):
        answer.append(0)
    for i in my_string:
        if ord(i) < 97:
            answer[ord(i) -65] = answer[ord(i) -65] + 1
        else:
            answer[ord(i) -71] = answer[ord(i) -71] + 1
    return answer

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {
    public int[] solution(String my_string) {
        int[] answer = new int[52];
        for (char c : my_string.toCharArray()) {
            if (c <= 'Z') {
                answer[c - 65]++;
            } else {
                answer[c - 'a' + 26]++;
            }
        }
        return answer;
    }
}